modules/jenkins.test.js

```diff
@@ -0,0 +1,111 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const jenkins = require("./jenkins");
+const fs = require("fs");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("jenkins", () => {
+    describe("configureCredentials", () => {
+        it("when url is empty then returns false", () => {
+            // Arrange
+            const url = "";
+            const username = "aaa";
+            const token = "aaa";
+            // Act
```

Comment:
Nitpick - new lines between each of the respective test steps (here and elsewhere in the file)